packages/build/src/js-transform.ts

```diff
@@ -29,8 +29,30 @@ import isWindows = require('is-windows');
 // minifying but not compiling?
 
 // Syntax and transform plugins for ES2015.
```

Comment:
Add note linking to bug about class plugin and mention its why we aren't using the preset.

Reply:
added